dataactvalidator/config/sqlrules/d31_detached_award_financial_assistance_2.sql

```diff
@@ -14,9 +14,12 @@ SELECT
     row_number,
```

Comment:
Can't make this comment above because GitHub hates me, but add what you're ignoring and why to the comments.